integration/samples/secondary/specs/es5-api.ts

```diff
@@ -22,6 +24,10 @@ describe(`@sample/secondary-lib`, () => {
 describe(`@sample/secondary-lib/sub-module`, () => {
 
   describe(`sub-module.es5.js`, () => {
+    let SECONDARY;
+    before(() => {
+      SECONDARY = require('../dist/esm5/sub-module.js');
+    });
```

Comment:
Prevents TypeScript compilation error for spec suite. Fails `before` hook in test run, if bundle files does not exist.